weblate/templates/trans/alert/brokenbrowserurl.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 <p>{% trans "This component is set up incorrectly." %}</p>
 
-<p>{% trans "The repository browser URL points to non existing locations:" %}</p>
+<p>{% trans "The repository browser URL points to a non-existing location:" %}</p>
```

Comment:
Either "URLs / locations" or "URL location" :)

Reply:
True, but the repository browser is actually a template and it evaluates to several locations.